spec.bs

```diff
@@ -1024,41 +1025,101 @@ partial interface Navigator {
 };
 </xmp>
 
-These updates have 2 triggers: the {{Navigator/updateAdInterestGroups()}} API, which updates all interest
-
-groups owned by the origin of the current frame, and post-auction updates, which, at the conclusion
-of {{Window/navigator}}.{{Navigator/runAdAuction()}}, updates the interest groups of all bidder origins that participated in the
-
-
-auction.
+The <dfn for=Navigator method>updateAdInterestGroups()</dfn> method steps are:
 
-<div algorithm="interest group update">
-
-Both update methods trigger the same algorithm, <dfn>interest group update</dfn> which is described below, and takes a list of owners to
-update called "owners":
+1. [=in parallel=] execute the algorithm [=interest group update=] using « [=this=]'s

```

Comment:
If this is a step in an algorithm, then I'd probably wrap this and the above dfn in a `<div algorithm></div>` block